### PR TITLE
Update DG to include Find feature implementation and PPP

### PR DESCRIPTION
**Pull request by KengXIII**
Updated PPP and DG section on findCommand.

### CODE REVIEW

**Review by nhjryan — CHANGES_REQUESTED** [review]
Might want to change the name of the PR! Looks a bit weird...

File: docs/team/kengxiii.md [hunk 1]
```diff
@@ -1,46 +1,66 @@
...
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
-
-* _{you can add/remove categories in the list above}_
+* PRs reviewed (with non-trivial review comments):
+[\#39](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/39),
+[\#74](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/74),
+[\#155](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/155),
+[\#159](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/159),
+* Took and cropped profile photos for [About-Us](https://ay2122s1-cs2103t-f12-3.github.io/tp/AboutUs.html) page.
```
Comment (ykwei7) [inline docs/team/kengxiii.md:66]: Maybe this portion can put under `Project Management` portion, and talk about setting up the aboutUs page

**Review by ykwei7 — APPROVED** [review]
LGTM, find feature is well explained and includes the 2 possible implementations we were deciding on!

File: docs/team/kengxiii.md (hunk 1 above)
Comment (KengXIII) [reply docs/team/kengxiii.md]: I took the suggestion and moved it to `Project Management` :)

**Pull request merged**